Mathlib/Util/GetAllModules.lean

```diff
@@ -39,18 +39,18 @@ def getAllFiles (git : Bool) (ml : String) : IO (Array System.FilePath) := do
     else do
       let all ← walkDir ml
       return all.filter (·.extension == some "lean"))
-  let files := (allModules.erase ml.lean).qsort (·.toString < ·.toString)
-  let existingFiles ← files.mapM fun f => do
+  let existingFiles ← (allModules.erase ml.lean).mapM fun f => do
```

Comment:
Does `filterMapM` exist?  And can it be used here?

Reply:
It exists, TIL. I'll take a look!

Reply:
Used it here, thanks for the pointer!